pkg/kf/commands/quotas/integration_test.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quotas
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	. "github.com/GoogleCloudPlatform/kf/pkg/kf/testutil"
+)
+
+// TestIntegration_Create creates a resourcequota, then tries to retrieve the quota info.
+// It finally deletes the quota.
+func TestIntegration_Create(t *testing.T) {
+	checkClusterStatus(t)
+	RunKfTest(t, func(ctx context.Context, t *testing.T, kf *Kf) {
+		quotaName := fmt.Sprintf("integration-quota-%d", time.Now().UnixNano())
+		defer kf.DeleteQuota(ctx, quotaName)
+
+		memQuantity := "500Gi"
+		cpuQuantity := "10"
+		routesQuantity := "100"
+
+		createQuotaOutput := kf.CreateQuota(ctx, quotaName,
+			"-m", memQuantity,
+			"-c", cpuQuantity,
```

Comment:
So I ran the tests after using a different value than `cpuQuanitity` but the test is still looking for it and they still passed. I think the assertions are getting tripped up with `10` vs `100`.

Reply:
thanks for catching that, i updated the test with new values with distinct digits